### PR TITLE
Add Color provider to es_ES locale

### PR DESCRIPTION
**Pull request by juananruiz**
Translating colors to Spanish and corrected accent mark in a name

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.idea
```
Comment (fzaninotto) [inline .gitignore]: please remove

File: src/Faker/Provider/es_ES/Color.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Faker\Provider\es_ES;
+
+class Color extends \Faker\Provider\Color
+{
+    protected static $safeColorNames = array(
+        'negro', 'marrón', 'verde', 'azul marino', 'oliva',
```
Comment (fzaninotto) [inline src/Faker/Provider/es_ES/Color.php]: please order alphabetically, to ease future additions / deletions

File: .gitignore (hunk 1 above)
Comment (juananruiz) [reply .gitignore]: OK, it's me first Pull Request in GitHub. 
Can i make something more for Spanish Translation?
Great Library!

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**